src/server/IlcSdk.ts

```diff
@@ -117,6 +135,25 @@ export default class IlcSdk {
         res.end(JSON.stringify(resData));
     }
 
+    private parseIntl(req: IncomingMessage) {
+        const intlParams = req.headers['x-request-intl'] as string | undefined;
+        if (intlParams === undefined) {
+            return null;
+        }
+
+        const paramsParts = intlParams.split(';');
```

Comment:
it's not important but maybe let's wrap it with `try/catch` and show error about correct syntax of the header?

Reply:
added